src/main/java/rx/Scheduler.java

```diff
@@ -43,6 +43,17 @@
  *    maintenance.
  */
 
+    /** 
+     * The tolerance for a clock drift in nanoseconds where the periodic scheduler will rebase. 
+     * <p>
+     * The associated system parameter, {@code rx.scheduler.drift-tolerance}, expects its value in minutes.
+     */
+    static final long CLOCK_DRIFT_TOLERANCE_NANOS;
+    static {
+        CLOCK_DRIFT_TOLERANCE_NANOS = TimeUnit.MINUTES.toNanos(
```

Comment:
I didn't inline it in case more parameters are asked for within this PR.

It is a question what is a reasonable tolerance for detecting drifts. One could argue it should be the function of the drift direction and the length of the period instead:

  - if now moves backwards by any number, do the correction,
  - if now moves ahead more than 1.5 times the period, do the correction.